cas-server-documentation/installation/SPNEGO-Authentication.md

```diff
@@ -109,7 +151,13 @@ Replace instances of `viewLoginForm` with `startSpnegoAuthenticate`, if any.
 
 ### Authentication Configuration
 
-Provide a JAAS `login.conf` file:
+Provide at least this CAS property :
+
+```properties
+cas.authn.spnego.jcifsServicePrincipal=HTTP/cas.example.com@REALM
```

Comment:
Also, please avoid referencing specific properties in the doc pages. You may instead say something like "Make sure you have specified the JCIFS Service Principal in the CAS configuration. Use this link to see relevant settings".